browser/resources/settings/shortcuts_page/commands.tsx

```diff
@@ -36,13 +36,24 @@ const SearchContainer = styled.div`
 `
 
 const SearchBox = styled.input`
-  border: 1px solid lightgray;
+  all: unset;
```

Comment:
is that needed? aren't we in a shadow DOM?

Reply:
Yup, it just removes some of the browser styles